bots/honkbot.py

```diff
@@ -73,7 +73,8 @@ async def on_message(self, message):
     async def on_command_error(self, ctx, error):
         self.logger.error(error)
         if isinstance(error, commands.CommandNotFound):
-            await ctx.send("Use !help to see a list of commands")
+            if ctx.invoked_with.isalnum():
+                await ctx.send(f"{ctx.invoked_with} is not a command. Use !help to see a list of commands")
```

Comment:
My assumption is that we won't be implementing any commands with punctuation, so this should catch stuff like !!! that people normally say and shouldn't trigger the bot.